datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -75,6 +75,15 @@ def __init__(self, *args, **kwargs):
         self.warnings = []
         self.metric_limiter = None
 
+        if datadog_agent.get_config('disable_unsafe_yaml'):
+            # monkey-patches unsafe pyyaml methods
+            try:
+                import yaml
+                from ..ddyaml import monkey_patch_pyyaml
+                monkey_patch_pyyaml()
+            except ImportError:
```

Comment:
why catch `ImportError`s here? Since `pyyaml` is in the `requirements.in` file of `datadog_checks_base`, it should always be there, or am I missing something?